Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'lotus-view',        '~> 0.4', require: false, github: 'lotus/view',
 gem 'lotus-model',       '~> 0.4', require: false, github: 'lotus/model',       branch: '0.4.x'
 gem 'lotus-helpers',     '~> 0.2', require: false, github: 'lotus/helpers',     branch: '0.2.x'
 
-platforms :ruby do
+platforms :ruby, :mswin, :mingw, :x64_mingw do
```

Comment:
Can you please remove Windows platform because atm we are not supporting it yet?